kuma/payments/utils.py

```diff
@@ -20,3 +26,53 @@ def recurring_payment_enabled(request):
     """Returns True if recurring payment is enabled for the user."""
     return (popup_enabled(request) and
             flag_is_active(request, RECURRING_PAYMENT_BETA_FLAG))
+
+
+def get_stripe_customer_data(stripe_customer_id, email, username):
```

Comment:
Where this function is getting used?

Reply:
This is shared part from https://github.com/potatolondon/kuma/pull/52 which is in internal review right now.